src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -477,6 +492,24 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
     ============================================================
+    _GetRestoreSettingsPerFramework
+    Generate items with framework specific settings.
+    ============================================================
+  -->
+  <Target Name="_GetRestoreSettingsPerFramework"
+    Returns="@(_RestoreSettingsPerFramework)">
+
+    <ItemGroup>
+      <_RestoreSettingsPerFramework Include="$([System.Guid]::NewGuid())">
```

Comment:
Can't you use the framework name here?
It'd make it more transparent and easier for debugging.